addons/purchase_requisition/migrations/10.0.0.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,67 @@
+---Fields in module 'purchase_requisition'---
+purchase_requisition / purchase.order.line      / quantity_tendered (float)     : DEL
+purchase_requisition / purchase.requisition     / exclusive (selection)         : DEL required: required, selection_keys: ['exclusive', 'multiple'], req_default: multiple
+purchase_requisition / purchase.requisition     / multiple_rfq_per_supplier (boolean): DEL
+# Nothing to do
+
+purchase_requisition / purchase.requisition     / type_id (many2one)            : NEW relation: purchase.requisition.type, required: required, req_default: function
+# Nothing to do
+
+purchase_requisition / purchase.requisition     / vendor_id (many2one)          : NEW relation: res.partner
+# Done: upgrade vendor with partner_id of purchase order
```

Comment:
You should leave it blank, as it's not required and you can have quotations from several vendors. This is a new feature for autocompleting new RfQs only.